utilities/manual/usermanual_en.html

```diff
@@ -548,14 +548,13 @@ <h1 id="SECTION2">3. Compiling a document</h1>
 <h2 id="SECTION22">3.1 Compiling</h2>
 <p>The easiest way to compile a document is to use the "Compile" command or the "Build&amp;View" command ("Compile" button - shortcut : F6). You can select the default command via the "Configure TeXstudio" dialog.<br>(You can also launch each command one by one in the "Tools" menu).<br> Note : the "Clean" command in the "Tools menu" allows you to erase the files (dvi, toc, aux...) generated by a LaTeX compilation (except the ps and pdf files).</p>
 <p><IMG src="compile_toolbar.png" border="1" alt="compile_toolbar">
-<p><b>Warning: all your files must have an extension and you can't compile an "untitled" file or a file with a space in its name</b>.</p>
+<p><b>Warning: all your files must have an extension and you can't compile a file with a space in its name</b>.</p>
```

Comment:
that is not true any more (for some time)
filename may contain spaces

Reply:
so we stay with a warning that all files must have an extension? or remove completely?